pkgs/development/python-modules/torchvision/prefetch.sh

```diff
@@ -5,7 +5,7 @@ set -eou pipefail
 
 version=$1
 
-linux_cuda_version="cu116"
```

Comment:
I haven't thoroughly looked at torch `-bin` packages before. Shouldn't this be synchronized with `cudaPackages.cudaVersion`? I also don't see any `autoPatchelfHook`s in `bin.nix`, so maybe I just misunderstand how this derivation works

Reply:
From what I have seen, the `-bin` derivations only download the appropriate binaries from torch website.
Those binaries come bundled with their own version of cuda, which does not need to be the same as the one on the system.
Hence, it should not be linked to the `cudaPackages.cudaVersion` and we don't need to use `autoPatchelfHook`.

As for which version of cuda to choose (the `cu117` in `prefetch.sh`), it is the one that torch ships by default with the latest version of each package.
https://pytorch.org/get-started/locally
i.e., the cuda version that yields to the canonical install command:
```sh
pip install torch torchvision torchaudio
```